src/libxrpl/protocol/Permissions.cpp

```diff
@@ -131,6 +131,19 @@ Permission::getGranularTxType(GranularPermissionType const& gpType) const
     return std::nullopt;
 }
 
+std::optional<std::reference_wrapper<uint256 const>> const
+Permission::getTxFeature(TxType txType) const
+{
+    auto const txFeaturesIt = txFeatureMap_.find(txType);
+    XRPL_ASSERT(
+        txFeaturesIt != txFeatureMap_.end(),
+        "ripple::Permissions::isDelegatable : tx exists in txFeatureMap_");
```

Comment:
bad edit, should be `Permission::getTxFeature : tx exists in txFeatureMap_` (we are allowed to rename asserts, if there's a good reason for it)